src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs

```diff
@@ -24,7 +26,7 @@ public RabbitMQClusterTransport(Topology topology, string connectionString, Queu
         /// <param name="port">The port of the node.</param>
         public void AddClusterNode(string host, int port = 5672)
         {
-            additionalHostnames.Add($"{host}:{port}");
+            additionalHosts.Add(new Tuple<string, int>(host, port));
```

Comment:
can't we use the simplified `(x,y)` tuple syntax in this project?

Reply:
[Computer says no](https://www.youtube.com/watch?v=TcRgyAkLeP4)

Reply:
http://gph.is/1UObEqG